### PR TITLE
Bump to version 0.24.3

### PR DESCRIPTION
**Pull request by DanielMorM**
### Platforms Impacted
- [X] iOS
- [X] macOS

### Description of changes

Bumping version number to 0.24.3

Included in this release:
- Respecting avatarOverrideStyle when setting TitleView style #1949 
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/fluentui-apple/pull/1952)

### CODE REVIEW

**Review by laminesm — APPROVED** [review]

**Review by mischreiber — APPROVED** [review]

**Review by mischreiber — CHANGES_REQUESTED** [review]
We bump the version as the last commit after any necessary changes have merged/cherry-picked into the release branch. Let's make sure we don't merge this until after that is done.

**Review by mischreiber — APPROVED** [review]

**Pull request merged**
